src/py/pyodide/console.py

```diff
@@ -236,7 +236,20 @@ def __call__(  # type: ignore[override]
 
 
 class ConsoleFuture(Future[Any]):
-    """A future with extra fields used as the return value for :py:class:`Console` apis."""
+    """
+    A future with extra fields used as the return value for :py:class:`Console` APIs.
+
+    Example:
+        >>> # doctest: +RUN_IN_PYODIDE
+        >>> from pyodide.console import Console
```

Comment:
How about this:
```suggestion
        >>> from pyodide.console import Console # doctest: +RUN_IN_PYODIDE
```
if this doesn't work we can go back to the `# doctest: +SKIP` approach. But I think it should work...

Reply:
I will make these changes